src/app/dashboard-widgets/recent-pipelines-widget/recent-pipelines-widget.component.html

```diff
@@ -56,7 +56,7 @@ <h3>Pipelines reflect integration and deployment stages that code goes through</
           <div class="f8-card__pipeline-column">
             <span class="{{buildconfig.iconStyle}} fa-spin" title="{{buildconfig.statusPhase}}" *ngIf="buildconfig.iconStyle === 'pficon-running'"></span>
             <span class="{{buildconfig.iconStyle}}" title="{{buildconfig.statusPhase}}" *ngIf="buildconfig.iconStyle !== 'pficon-running'"></span>
-            <a id="spacehome-pipelines-title" [routerLink]="[contextPath, buildconfig.labels['space'], 'create', 'pipelines']"
+            <a id="spacehome-pipelines-title" [routerLink]="['/', buildconfig.namespace, buildconfig.labels['space'], 'create', 'pipelines']"
```

Comment:
The namespace isn't always the correct choice. For example, when the user is 'jkang@redhat.com' and the namespace is 'jkang', the link will not work.

Reply:
Best case is there's a value we can use from the buildconfig, preferably the full url, otherwise the Contexts service should be able to give us a username.

Reply:
Hmm good call. Since it's a user's own pipelines, would it be acceptable here to subscribe to `UserService.loggedInUser` and use the username in the link here?

Reply:
That sounds like it would work. The Contexts service also provides the current user. I wonder which should be used, hah...

Reply:
Ah that sounds good too.